scripts/v.db.join/v.db.join.html

```diff
@@ -111,8 +111,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 Markus Neteler
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 107 

`<a href="v.db.update.html">v.db.update</a><br>`

should be 

`<a href="v.db.update.html">v.db.update</a>,`.

Reply:
Fixed as for `scripts/v.db.addcolumn/v.db.addcolumn.html` (see above)